tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<p>I would say:</p><p> </p><p>"SPSA Gradients compute the gradients...."</p><p> </p><p>and <code>epsilon</code> instead of Epsilon. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1463/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>